### PR TITLE
Theunis build landing page

### PR DESCRIPTION
**Pull request by TheunisK**

### CODE REVIEW

File: src/client/styles/landingPage.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    color: #6A5441;
+}
+
+div > h3 {
+    font-size: 2rem;
+    color: rgb(12, 134, 12);
+    font-weight: bold;
+}
+
+div > h3:hover {
```
Comment (hannah68) [inline src/client/styles/landingPage.css:65]: nice css Theunis.

**Review by hannah68 — APPROVED** [review]

**Review by Can-Tina — APPROVED** [review]

File: src/client/App.jsx [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
+// import { Routes, Route } from 'react-router-dom';
 import './App.css';
 
+import LandingPage from './pages/landingPage/LandingPage';
+
 function App() {
   return (
     <div className="App">
-      <p>Hello, world!</p>
+      <LandingPage />
```
Comment (glowkeeper) [inline src/client/App.jsx:10]: Oooo - looking forward to seeing it :)

**Review by glowkeeper — APPROVED** [review]
Approved

File: src/client/styles/landingPage.css (hunk 1 above)
Comment (TheunisK) [reply src/client/styles/landingPage.css:65]: Thanks Hannah! Not as good as your...

**Pull request merged**